ldm/dream/readline.py

```diff
@@ -17,8 +17,11 @@
 try:
     import readline
     readline_available = True
-except:
-    readline_available = False
+except (ImportError,ModuleNotFoundError):
+    try:
+        import pyreadline3
+    except (ImportError,ModuleNotFoundError):
+        readline_available = False
```

Comment:
Unnecessary. The `pyreadline` docs say "pyreadline3 installs a file under the name 'readline.py' in site-packages"

Reply:
Ok, I'll delete this logic and trust that it works.